File: src/main/java/me/pafias/gambling/Card.java
package me.pafias.gambling;

public record Card(Suit suit, Rank rank) {
    @Override
    public String toString() {
        // Formats the card for display, e.g., "[♥ K]"
        return "&f[" + suit.getIcon() + " " + rank.getDisplayName() + "&f]";
    }
}

File: src/main/kotlin/me/pafias/gambling/Card.kt
package me.pafias.gambling

data class Card(val suit: Suit?, val rank: Rank?) {
    override fun toString(): String {
        // Formats the card for display, e.g., "[♥ K]"
        return "&f[" + suit!!.icon + " " + rank!!.displayName + "&f]"
    }
}